.circleci/config.yml

```diff
@@ -70,17 +70,17 @@ jobs:
   build_for_tests:
     executor: main-executor
     environment:
-      - FEDERATION: "true"
```

Comment:
Standardizing on the format listed here: https://circleci.com/docs/2.0/env-vars/#setting-an-environment-variable-in-a-job